README.md

```diff
@@ -1,2 +1,41 @@
-# conduit-connector-amazon-sqs
-Conduit connector for Amazon SQS 
+# Conduit Connector for Amazon SQS 
+[Conduit](https://conduit.io) for [Amazon SQS](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/welcome.html).
+
+## How to build?
+Run `make build` to build the connector.
+
+
+## Source
+The source connector pulls data from the Amazon SQS Queue. As messages come in, the source connector grabs a single message from the Amazon SQS Queue, formats it for the destination connector as a new record, and sends it. The Message Body of the SQS Message is formatted into a record's payload, and the Message Attributes are passed as record metadata. After the record has been acknowledged, the source connector deletes the message from the Amazon SQS Queue.
+
+
+## Destinaton
+
+
+The destination connector splits incoming records into batches of 10 and pushes them to the Amazon SQS Queue. Any fields defined in the metadata of the record will be passed as Message Attributes, and the json encoding of the record will be passed as the Message Body.
```

Comment:
Nitpick: I believe that it's technically more correct to say that incoming records are batched, and not split.